src/interfaces/modular/Clustering.i

```diff
@@ -18,12 +18,16 @@
 /* Remove C Prefix */
 %rename(DistanceMachine) CDistanceMachine;
 %rename(Hierarchical) CHierarchical;
+%rename(KMeansBase) CKMeansBase;
```

Comment:
base class needs to be exposed to the interfaces? Users will never use it right?